doc/en/permissions.md

```diff
@@ -0,0 +1,65 @@
+# Managing permissions for sessions
+
+By default, users can only see and invalidate their own sessions. Not even an administrator can see other users' sessions. This is meant to protect the privacy of all users.
+
+Specific projects may wish to allow a some users to view and manage the sessions of other users.
+
+## A note about privacy
```

Comment:
Normally, we don't discuss legal/ethical implication in technical doc, but it seemed especially relevant in this scenario.

Reply:
Agree